x/gamm/pool-models/stableswap/amm.go

```diff
@@ -265,14 +265,19 @@ func targetKCalculator(x0, y0, w, yf osmomath.BigDec) osmomath.BigDec {
 
 // $$k_{iter}(x_f) = -x_{out}^3 + 3 x_0 x_{out}^2 - (y_f^2 + w + 3x_0^2)x_{out}$$
 // where x_out = x_0 - x_f
+// iterKCalculator returns a constantly increasing function to use in cfmm binary solver. Uses horners method https://www.geeksforgeeks.org/horners-method-polynomial-evaluation/
+// x0 - initial reserve of token X
+// w - w is a reserve of assets we do not touch when swapping (ex: in 3 asset pool {x, y, z}, if we swap x for y, `w` would include reserves of token z, because we do not touch it during swap). w = 0 in 2 asset pools
+// yf - y_final. Initial reserves of y plus added liquidity.
```

Comment:
```suggestion
// w - w is a reserve of assets we do not touch when swapping (ex: in 3 asset pool {x, y, z},
// if we swap x for y, `w` would include reserves of token z, because we do not touch it during swap). w = 0 in 2 asset pools
// yf - y_final. Initial reserves of y plus added liquidity.
```